list/README.md

```
@@ -599,7 +599,8 @@ Initially created by [Marko](https://markodenic.com) at [Web Development Resourc
 | [10015.io](https://10015.io/)                                                    | All-in-one toolbox to make life easier. Includes CSS generators, AI palette generator, and much more.                     |
 | [ToolsChimp.com](https://toolschimp.com/)                                        | 1000+ Free Online tools. No Signup, No Paywall. Color Converters, Image Tools, Video Tools, etc.,                         |
 | [volumeshaderbm.org](https://volumeshaderbm.org)                                 | A browser-based volume shader benchmark with interactive kernels and reproducible presets.                                |
-| [Format JSON Online](https://formatjsononline.com)                                 | A Complete Toolkit for JSON & Data Processing.|
+| [Format JSON Online](https://formatjsononline.com)                               | A Complete Toolkit for JSON & Data Processing.                                                     |
+| [webtoolz](https://webtoolz.dev)                                                 | JSON studio with AI-powered fix for broken JSON. Also formats, validates, minifies. Browser-only.  |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)